src/Command/InstallCommand.php

```diff
@@ -54,9 +54,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
             $io->success('Environment successfully installed.');
             $io->info(
```

Comment:
I think it would be more readable to keep these two pieces of information in dedicated messages. What do you think about the snippet below?
```
$io->note('The "origami database:details" command will help you to set up the connection in your project.');
```

Reply:
Why not, I update it